core/src/main/scala/org/apache/spark/sql/SnappyDDLParser.scala

```diff
@@ -699,6 +708,41 @@ private[sql] case class TruncateTable(ifExists: Any,
   }
 }
 
+private[sql] case class AlterTableAddColumn(tableIdent: TableIdentifier,
+   addColumn: StructField) extends ExecuteCommand {
+
+  override def run(session: SparkSession): Seq[Row] = {
+    val snc = session.asInstanceOf[SnappySession]
+    val catalog = snc.sessionState.catalog
+    snc.alterTable(catalog.newQualifiedTableName(tableIdent), true, addColumn)
+    Seq.empty
+  }
+}
+
+private[sql] case class AlterTableDropColumn(
+  tableIdent: TableIdentifier, column: String) extends ExecuteCommand {
+
+  override def run(session: SparkSession): Seq[Row] = {
+    val snc = session.asInstanceOf[SnappySession]
+    val catalog = snc.sessionState.catalog
+    val plan = try {
+      snc.sessionCatalog.lookupRelation(tableIdent)
```

Comment:
Why drop column is doing a lookup of relation but add column is not? Any reason for this difference? Actually you are doing this in the actual alterTable in the snappysession class below for both cases. You can just remove from here. Isn't it?